app/assets/src/components/views/report/ReportTable/DetailCells.jsx

```diff
@@ -0,0 +1,58 @@
+import React from "react";
```

Comment:
Current convention is to keep folders in snake case, so: `ReportTable` -> `report_table`